examples_tests/60.ClusteredRendering/intersection_record.glsl

```diff
@@ -0,0 +1,58 @@
+#ifndef _INTERSECTION_RECORD_H_
```

Comment:
put in nbl/builtin/glsl/ext/clusteredlighting/ and add appropriate #ifdef/#define guards and nbl_glsl_ext_ClusteredLighting_ prefixes to declarations and variable names